CIP-0129/README.md

```diff
@@ -0,0 +1,167 @@
+---
+CIP: 129
+Title: Governance Identifiers
+Status: Proposed
+Category: Tools
+Authors:
+  - Ashish Prajapati <ashish@strica.io>
+Implementors: N/A
+Discussions:
+  - https://github.com/cardano-foundation/CIPs/pull/845
+  - https://github.com/cardano-foundation/CIPs/pull/857
+Created: 2024-07-15
+License: CC-BY-4.0
+---
+
+## Abstract
+
+This specification describes the structure and the bech32 format of the identifiers for DRep, CC keys, and Gov Actions
+
+## Motivation: why is this CIP necessary?
+
+Cardano Conway era introduces new credentials for different actors, and introduces proposal procedures for governance actions. It is required to be able to find, link, share these identifiers in bytes form across the tooling and community.
+
+## Specification
+
+### Introduction
+We define a bytes representation for the various credentials and identifiers along with the their bech32 prefixes in this CIP. Taking inspiration from the Cardano addresses bytes format, we define an 8 byte header and a payload to define the key, which look similar to the reward address byte form but with a new specification and using the governance credentials.
```

Comment:
What is the rationale for not just reusing how the Address headers as defined via [CIP-19](https://github.com/cardano-foundation/CIPs/tree/master/CIP-0019#binary-format)?

Why make a new definition for a header?

Reply:
1 - gov identifiers are not addresses
2 - The current address spec header does not have enough space to accommodate these many identifiers